src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -102,6 +105,21 @@ class SidebarPanels extends React.Component<Props, State> {
         this.setState({ isInitialized: true });
     }
 
+    componentDidUpdate(prevProps: Props): void {
+        const { location, onVersionChange }: Props = this.props;
+        const { location: prevLocation }: Props = prevProps;
```

Comment:
Are the `: Props` explicit types needed here? I would think the type would be inferred from the variables.

Reply:
No, they are not needed. I copy-pasted from `Sidebar.js`. Deleted!